sentry-android-core/src/main/java/io/sentry/android/core/ScreenshotEventProcessor.java

```diff
@@ -51,10 +52,17 @@ public ScreenshotEventProcessor(
     application.registerActivityLifecycleCallbacks(this);
   }
 
+  public void setCurrentActivity(@NonNull Activity activity) {
+    if (currentActivity != null && currentActivity.get() == activity) {
+      return;
+    }
+    currentActivity = new WeakReference<>(activity);
+  }
```

Comment:
L: any reason to move the code?

Reply:
I don't know how is the lint set in this project, but I saw everywhere the public methods are before private.